### PR TITLE
Short bonsai report link & update docs with deprecation warning to module types reference pages

### PR DESCRIPTION
**Pull request by shumailxyz**
**What this PR does / why we need it**:
- Adds a deprecation warning to all module type reference pages .
- fix a typo in action docs
- change bonsai issue report link to shinny new shortlink ;) 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by vvagaytsev — APPROVED** [review]
Nice! 👍

File: static/docs/templates/module-type.hbs [hunk 1]
```diff
@@ -5,6 +5,10 @@ tocTitle: "`{{{name}}}`"
 
 # {{{frontmatterTitle}}}
 
+{% hint style="warning" %}
```
Comment (Orzelius) [inline static/docs/templates/module-type.hbs:8]: A bit unrelated but I wonder if markdown supported syntax could be used so it also renders when viewing on gh

```
> **Warning** text goes here
> or on the next line here
```
> **Warning** text goes here
> or on the next line here

**Review by Orzelius — DISMISSED** [review]

File: static/docs/templates/module-type.hbs (hunk 1 above)
Comment (shumailxyz) [reply static/docs/templates/module-type.hbs:8]: I am not sure if Gitbook would pick that properly.

**Review by vvagaytsev — APPROVED** [review]
🚀

**Review by Orzelius — APPROVED** [review]
I'm glad we got the short link in

**Pull request merged**